tests/parity/toolcalling/capture_toolcalling_outputs.py

```
@@ -414,6 +439,126 @@ def merge_into_fixtures(
+
+def _run_all_in_container(
+    container: str, impl: str, requests: list[dict]
+) -> dict[str, dict]:
+    """Copy the worker bundle into `container` and run every request there once.
+
+    Returns {key: {calls, normal_text, error}}. The engine import happens a
+    single time inside the worker, so the whole batch shares one heavy startup.
+    Works for devcontainers and bare engine images alike (nothing is assumed
+    about a mounted repo — the bundle is copied in).
+    """
+    dest = "/tmp/parity_worker"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/parity_worker"

(S108)